pages/templates/pages/group_index_page.html

```diff
@@ -42,6 +35,14 @@ <h4 class="mb20 mt0">No group in your area?</h4>
   </div>
 </div>
 
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12 visible-xs f5f5f5-bg pa20">
```

Comment:
Looks like standard grid now 😄 

One thing I'm not sure about - should `visible-xs` class be on the whole container?

Reply:
Oh interesting, this format is deprecated: http://getbootstrap.com/docs/3.3/css/#responsive-utilities-classes

Should be `visible-xs-block` now?